\section{Discussion}
BSC N-fixation has long been attributed to heterocystous
\textit{Cyanobacteria} and molecular surveys of BSC \textit{nifH} gene content
have been consistent with this hypothesis finding cyanobacterial \textit{nifH}
types to be numerically dominant in \textit{nifH} gene libraries from BSC
\citep{Yeager,14766579,Yeager_2012}. However, $^{15}$N$_{2}$ DNA-SIP revealed
non-cyanobacterial microbes fixed N$_{2}$ in early successional BSC samples.
After 2 and 4 incubation days in the presence or absenece of $^{15}$N$_{2}$
in microcosm headspace, DNA from early successional BSC samples was collected
and separated by bouyant density in CsCl density gradients.
Heavy CsCl gradient fractions from gradients with $^{15}$N-labelled DNA were
different in phylogenetic membership/structure than heavy fractions with
unlabeled DNA (Figure X).  Further, heavy gradient fractions clustered by DNA
type (labeled or unlabeled) (Figure X). Therefore, headspace $^{15}$N$_{2}$ in
early successional BSC microcosms was incorporated into biomass, and, the
specific OTUs that incorporated $^{15}$N (from $^{15}$N$_{2}$) into biomass
would be enriched in labeled gradient heavy fractions relative to control.
\textit{Proteobacteria} and \textit{Clostridiaceae} represented most OTUs
enriched in DNA from labeled gradient heavy fractions relative to control as
revealed by a robust statistical framework for quantifying and evaluating
differential OTU abundance in microbiome studies (CITE McMurdie), Additionally,
\textit{Proteobacteria} and \textit{Clostridiaceae} represented OTUs that most
strongly responded to $^{15}$N among all responders. SIP places focus upon
organisms based on isotope incorporation and has the ability to detect activity
by low abundance members of the community.  DNA from OTUs that incopororate
$^{15}$N into their biomass moves towards the heavy end of the CsCl gradient
and therefore OTUs in ``labeled'' DNA are enriched in the full data pool
relative to bulk DNA.  Phylum-level taxonomic annotations of
$^{15}$N-responsive OTUs (i.e.  \textit{Firmicutes} and
\textit{Proteobacteria}) are enriched in the DNA-SIP data relative to
environmental data (Figure~\ref{fig:study_phy_dist}).

It is possible that PCR-driven molecular surveys of \textit{nifH}
gene content have been biased. We propose three mechanisms that could bias
\textit{nifH} clone libraries against heteroptophic diazotrophs. First,
polyploidy in \textit{Cyanobacteria} \citep{Griese_2011} would inflate 
the representation cyanobacteria in community DNA beyond their cell number
representation. Second, \textit{nifH} PCR primers could be biased against
heterotrophic diazotrophs. In general the nifH PCR primers used by
\citet{Yeager,14766579,Yeager_2012} (``19F'' and ``nifH3'') for the first round
of nested PCR have broad specificity and display at least 86\% \textit{in
silico} coverage for \textit{Proteobacteria, Cyanobacteria} and ``Cluster III''
(which includes clostridial \textit{nifH}) reference \textit{nifH} sequences
\citep{Gaby_2012}. In the second round of the nested PCR protocol
\citep{Yeager,14766579,Yeager_2012}, primer ``nifH11'' is` biased against
``Cluster III'' (50\% \textit{in silico} coverage of reference \textit{nifH}
sequences), \textit{Proteobacteria} (79\% coverage) and
\textit{Cyanobacteria} (67\% coverage), and, primer ``nifH22'' matches
\textit{Proteobacteria}, \textit{Cyanobacteria} and ``Cluster III'' reference
sequences poorly (16\%, 23\% and 21\% \textit{in silico} coverage,
respectively) \citep{Gaby_2012}.  Unfortunately, it is difficult to assess or
quantify this bias (in either direction) without knowing the \textit{nifH} gene
content \textit{de novo}. Third, heterocysts (the specialized N-fixing cells
along the trichome of filamentous heterocystous \textit{Cyanobacteria} such as
\textit{Nostoc} and \textit{Scytonema}) may be overrepresented with respect to
non-cyanobacterial diazotrophs because heterocysts make up a fraction of cells
along a trichome and even the non-heterocyst cells in the trichome will possess
the \textit{nifH} gene. As a result of polyploidy and the frequency of
heterocysts in a cyanobacterial filament, the ratio of cyanobacterial
to heterotroph \textit{nifH} gene copies may be on the order of
10$^{2}$-10$^{3}$ higher than the ratio of heterocysts to heterotrophic
diazotroph cells. Regardless, our results suggest that BSC N-fixation may
include a significant non-cyanobacterial component that requires further
assessment across a more comprehensive sampling of BSC types.

We did not observe evidence for N-fixation by heterocystous
\textit{Cyanobacteria} in the early successional BSC samples used in this
study. One possible explanation for our results is that the early successional
BSC samples used in this study possessed too few heterocystous
\textit{Cyanobacteria} to statistically evaluate their $^{15}$N-incorporation.
Indeed, only 0.29\% of sequences from this study's DNA-SIP 16S rRNA gene
sequence libraries were from heterocystous \textit{Cyanobacteria} (see results)
as opposed to 15\% and 23\% of total sequences in the \citet{Steven_2013} and
\citet{Garcia_Pichel_2013} data, respectively.  Nonetheless, we would still
expect even low abundance diazotrophs to show evidence for
$^{15}$N-incorporation, provided sequence counts were not too sparse in heavy
fractions. The OTUs defined by selected heterocystous \textit{Cyanobacteria}
sequences presented in \citet{Yeager}, however, all fall below the sparsity
threshold used in our analysis (see methods). Given the sparsity of
heterocystous \textit{Cyanobacteria} sequences in the DNA-SIP data set, it is
not possible to assess whether heterocystous \textit{Cyanobacteria}
incorporated $^{15}$N during the incubation. It should be noted that "light"
and in particular "sub-biocrust" samples possess much less heterocystous
\textit{Cyanobacteria} in general (Figure~\ref{fig:het_steven}) so the samples
used in this study are not necessarily unrepresentative of typical poorly
developed BSC simply because they are lacking heterocystous
\textit{Cyanobacteria}. 

The OTUs that did appear to incorporate $^{15}$N during the incubation were
predominantly \textit{Proteobacteria} and \textit{Firmicutes}. The
\textit{Proteobacteria} OTUs for which $^{15}$N-incorporation signal was
strongest all shared high sequence identity ($>=$98.48\%) with 16S sequences
from cultivars in genera with known diazotrophs (Table~\ref{tab:LTP_blast}).
The \textit{Firmicutes} that displayed signal for $^{15}$N-incorporation
(predominantly \textit{Clostridiaceae}) were not closely related to any
cultivars (Table~\ref{tab:LTP_blast}, Figure~\ref{fig:clost_tree}). These BSC
\textit{Clostrodiaceae} diazotrophs represent a gap in culture collections. As
culture-based ecophysiological studies have proven useful towards explaining
ecological phenomena in BSC 16S rRNA gene sequence libraries
\citep{Garcia_Pichel_2013}, it would seem that these putative
\textit{Clostridiaceae} diazotrophs would be prime candidates for targeted
culturing efforts. Assessing the physiological response of these diazotrophic
\textit{Clostridiaceae} to temperature would be useful for predicting how
climate change will affect the BSC nitrogen budget.

Although too undersampled in the environmental data sets to reach statistical
conclusions, $^{15}$N-responsive OTUs were found more often in sub-crust or
or early successional BSC samples (Figure~\ref{fig:trees} and
Figure~\ref{fig:rspndr_dist}).  This result generates some hypotheses that are
counter to prior discussions regarding BSC diazotroph temporal dynamics.
Specifically, the succession of BSC may not be marked by the \textit{emergence}
of diazotrophs in BSC but rather the \textit{transition} of the diazotroph
community from heterotroph dominance to cyanobacteria.  Additionally,
sub-crust soil may contribute significantly to the N budget in arid ecosystems.

We propose that fast-growing heterotrophic diazotrophs such as
\textit{Clostridiaceae} may be BSC pioneers. \textit{M. vaginatus}
accumulate compatible solutes such as trehalose and sucrose as osmoprotectants
during dessication \citep{Rajeev_2013}. Additionally, although not demonstrated specifically with \textit{M. vaginatus}, microorganisms can rapidly excrete 
compatible solutes upon wetting \citep{Poolman_1998}. Many \textit{Clostridiaceae} have a saccharolytic metabolism \citep{Wiegel_2006} and \textit{Clostridiaceae} isolates have been shown to utilize trehalose and/or sucrose (CITE examples). Further, \textit{Clostridiaceae} isolates are fast-growing (doubling times typically  between 30 min and 3 hr when grown on monosaccharides in culture \citep{Wiegel_2006}). Upon wetting, the early successional BSC environment may become rapidly rich in compatible solutes excreted by \textit{M. vaginatus}. This boom-bust cycle would favor fast-growing microorganisms such as \textit{Clostridiaceae} that can double
rapidly and also fix N.

Rarefaction curves of all samples from \citet{Steven_2013} and
\citet{Garcia_Pichel_2013} are still sharply increasing especially for
sub-crust samples (Figure~\ref{fig:rarefaction}). Parametric richness estimates
of BSC diversity indicate the \citet{Steven_2013} and
\citet{Garcia_Pichel_2013} sequencing efforts recovered on average 40.5\% (sd.
9.99\%) and 45.5\% (sd.  11.6\%) of existing 16S OTUs from samples (inset
Figure~\ref{fig:rarefaction}), respectively. Further, the \citet{Steven_2013}
and \citet{Garcia_Pichel_2013} sequence collections only share 57.6\% of total
OTUs found in at least one of the studies. In fact, this study shares more OTUs
with \citet{Steven_2013}, 62.4\% of OTUs in the combined data, than the
\citet{Steven_2013} study shares with \citet{Garcia_Pichel_2013}.  Therefore,
is not alarming that few of the $^{15}$N-responsive OTUS were found by
\citet{Garcia_Pichel_2013} and \citet{Steven_2013}. Even next-generation
sequencing efforts of BSC 16S rRNA genes have only shallowly sampled the full
diversity of BSC microbes.  

\subsection{Conclusion}
Heterocystous \textit{Cyanobacteria} are key contributors to the BSC N-budget,
but, the $^{15}$N-responsive OTUs found in this study and the \textit{nifH}
gene sequences from \citet{Steppe_1996} in addition to the N-fixation rate data
presented by \citet{15643930} suggest there may be significant
non-cyanobacterial BSC diazotrophs specifically within the
\textit{Clostrideaceae} and \textit{Proteobacteria}. It seems clear that
heterocystous \textit{Cyanobacteria} increase in abundance with BSC age
\citep{14766579}. It is less clear if this transition marks the emergence of
diazotrophy versus a re-structuring of the BSC diazotroph community from one
dominated by \textit{Firmicutes} and \textit{Proteobacteria} to one
predominantly heterocystous \textit{Cyanobacteria}. DNA-SIP is a valuable tool
in the molecular microbial ecologist's toolbox for identifying members of
microbial community functional guilds \citep{17446886}. PCR-based surveys of
diagnostic marker genes and DNA-SIP are both used to connect microbial
phylogenetic types to microbial activities, but they occupy a non-overlapping
set of strengths and weaknesses. DNA-SIP does not focus on a specific diagnostic
marker but does identify \textit{active} players in the studied process (i.e. 
N-fixation). Combined these tools can powerfully reveal connections between
ecosystem membership/structure and function. Here we supplement previous
surveys of BSC \textit{nifH} diversity, a diagnostic marker PCR-driven
approach, with $^{15}$N$_{2}$ DNA-SIP, While we do not confirm previous
results, we expand knowledge of BSC diazotroph diversity.  Predicting BSC
N-fixation with respect to climate change, althered precipitation regimes and
physical disturbance requires a careful accounting of diazotrophs including
non-cyanobacterial types. 
