\section{Methods}
\subsection{Field sites}
\subsection{Soil crust incubation}
\subsection{DNA extraction}
DNA from each sample was extracted using a MoBio PowerSoil DNA Isolation Kit (following manufacturer’s protocol, but substituting a 2 minute bead beating for the vortexing step), and then gel purified. Extracts were quantified using PicoGreen nucleic acid quantification dyes (Molecular Probes). 
\subsection{DNA-SIP}
 Gradient density centrifugation of DNA was undertaken in 6 mL polyallomer centrifuge tubes in a TLA-110 fixed angle rotor (both Beckman Coulter) in CsCl gradients with an average density of 1.725 g mL-1.  Average density for all prepared gradients was checked with an AR200 refractometer before runs. Between 2.5- 5 μg of DNA extract was added to the CsCl solution, and gradients were run under conditions of 20°C for 67 hours at 55,000 rpm (Lueders et al., 2004). Centrifuged gradients were fractionated from bottom to top in 36 equal fractions of 100 μL, using a displacement technique similar to Manefield et al. (2002). The density of each fraction was determined using a refractometer. DNA in each fraction was desalted through four washes with 300 μL TE per fraction. 
\subsection{PCR, library normalization and DNA sequencing}
Bacterial and archaeal 16s rRNA genes from each fraction were quantified through real-time PCR, using primers Ba519f/Ba907r (Stubner, 2002).  Each 25 μL reaction contained 1X Quantifast SYBR Green Master Mix (Qiagen), 0.3 μM of each primer, and 1 μL of a 1:100 dilution of fraction DNA. Thermal cycling occurred with an initial denaturation step of 10 minutes at 95°C, followed by 40 cycles of amplification (15s at 95°C, then 60s at 60°C). After each run, a melt curve was measured and recorded between 60°C and 95°C. Quantification was achieved through use of a dilution series (108-101 copies/μL) from nearly full length 16s rRNA gene amplicons from pure culture DNA of K. pneumoniae. 

Barcoded PCR of bacterial and archaeal 16s rRNA genes, in preparation for 454 Pyrosequencing, was carried out using primer set 515F/806R \cite{21349862}. The primer 806R contained an 8 bp barcode sequence, a "TC" linker, and a Roche 454 B sequencing adaptor, while the primer 515F contained the Roche 454 A sequencing adapter. Each 25 $\mu$L reaction contained 1x PCR Gold Buffer (Roche), 2.5 mM MgCl2, 200 $\mu$M of each of the four dNTPs (Promega), 0.5 mg/mL BSA (New England Biolabs), 0.3 $\mu$M of each primers, 1.25 U of Amplitaq Gold (Roche), and 8 $\mu$L of template. Template for each sample was added at normalized amounts in an attempt to prevent chimera formation, and each sample was amplified in triplicate. Thermal cycling occurred with an initial denaturation step of 5 minutes at 95°C, followed by 40 cycles of amplification (20s at 95°C, 20s at 53°C, 30s at 72°C), and a final extension step of 5 min at 72°C. Triplicate amplicons were pooled and purified using Agencourt AMPure PCR purification beads, following manufacturer’s protocol. Once cleaned, amplicons were quantified using PicoGreen nucleic acid quantification dyes (Molecular Probes) and pooled together in equimolar amounts. Samples were sent to the Environmental Genomics Core Facility at the University of South Carolina (now Selah Genomics) to be run on a Roche FLX 454 pyrosequencing machine. 

\subsection{Data Analysis} 

\subsubsection{Sequence Quality Control} Sequences were initially screened by maximum expected errors at a specific read length threshold \cite{23955772} which has been shown to be as effective as denoising 454 reads with respect to removing pyrosequencing errors. Specifically, reads were first truncated to 230 nt (all reads shorter than 230 nt were discarded) and any read that exceeded a maximum expected error threshold of 1.0 was removed. After truncation and max expected error trimming, 91\% of original reads remained. The first 30 nt representing the forward primer and barcode on high quality, truncated reads were trimmed. Remaining reads were taxonomically annotated using the "UClust" taxonomic annotation framework in the QIIME software package \cite{20383131, 20709691} with cluster seeds from Silva SSU rRNA database \cite{17947321} 97\% sequence identity OTUs as reference (release 111Ref). Reads annotated as "Chloroplast", "Eukaryota", "Archaea", "Unassigned" or "mitochondria" were culled from the dataset. Finally, reads were aligned to the Silva reference alignment provided by the Mothur software package \cite{19801464} using the Mothur NAST aligner \cite{16845035}. All reads that did not appear to align to the expected amplicon region of the SSU rRNA gene were discarded. Quality control parameters removed 34716 of 258763 raw reads.

\subsubsection{Sequence Clustering}
Sequences were distributed into OTUs using the UParse methodology \cite{23955772}. Specically, cluster seeds were identified using USearch with a collection of non-redundant reads sorted by count as input. The sequence identity threshold for establishing a new OTU centroid was 97\%. After initial cluster centroid selection, select 16S rRNA sequences trimmed to the same 16S position as the other centroids from \citet{Yeager} were added to the centroid collection. Specifically, \citet{Yeager} Colorado Plateau or Moab, Utah sequences were included. Denovo centroid sequences that matched selected \citet{Yeager} sequences with greater than to 97\% sequence identity were subsequently removed from the centroid collection. With USearch/UParse, potential chimeras are identified during OTU centroid selection and are not allowed to become cluster centroids effectively removing chimeras from the read pool. All quality controlled reads were then mapped to cluster centroids at an identity threshold of 97\% again using USearch. 93.2\% of quality controlled reads could be mapped to centroids. Unmapped reads do not count towards sample counts and are essentially removed from downstream analyses. The USearch software version for cluster generation was 7.0.1090.

\subsubsection{Phylogenetic Tree}
The alignment for the "\textit{Clostridiaceae}" phylogeny was created using SSU-Align which is based on Infernal \cite{24008419, 19307242}. Columns in the alignment that were not included in the SSU-Align covariance models or were aligned with poor confidence (less than 95\% of characters in a position had posterior probability alignment scores of at least 95\%) were masked for phylogenetic reconstruction. Additionally, the aligment was trimmed to coordinates such that all sequences in the alignment began and ended at the same positions. The "\textit{Clostridiaceae}" tree included all top BLAST hits (parameters below) for $^{15}$N responders in the Living Tree Project database \cite{Yarza_2008} in addition to BLAST hits within an sequence identity threshold of 97\% to $^{15}$N responders from the Silva SSURef\_NR SSU rRNA database \cite{17947321}. Only one SSURef\_NR115 hit per study per OTU ("study" was determined by "title" field) was selected for the tree. FastTree \cite{20224823} was used to build the tree and split support values are SH-like scores reported by FastTree.

\subsubsection{BLAST Searches}
BLAST searches were done with the "blastn" program from BLAST+ toolkit \cite{20003500} version 2.2.29+. Default parameters were always emplyed and the BioPython \cite{19304878} BLAST+ wrapper was used to invoke the blastn program. Pandas \cite{citeulike:11241428} and dplyr \cite{dplyr} were used to parse and munge BLAST output tables.  

\subsubsection{Identifying OTUs that Inocorporated $^{15}$N into their DNA}
DNA with $^{15}$N as opposed to $^{14}$N has greater bouyant density CITE. Therefore, the placement of diazotroph population DNA in a CsCl gradient will be different depending on if the population grew in the presence of $^{15}$N$_{2}$ or $^{14}$N$_{2}$. Specifically, if the population incorporates $^{15}$ N the distribution of its DNA will shift towards heavier regions of a CsCl gradient relative to the same population that incorporates $^{14}$N. To identify populations (or in this case a proxy to populations, OTUs) that fix N, two microcosms are set up. One, the control, with $^{14}$N$_{2}$ in the headspace and the other with $^{15}$N$_{2}$. By comparing the relative abundance profiles of OTUs across CsCl gradients containing DNA from each microcosm OTUs which increase in abundance in the heavy fractions of $^{15}$N headspace incubated DNA relative to control can be identified. These OTUs have putatively incorporated N into their DNA and are potentially diazotrophs. For example, DNA-SIP with $^{15}$N as the stable isotope tracer has been successfully utilized to identify free living diazotrophs in agricultural soil CITE. We refer to CsCl gradients that received DNA from incubations with $^{15}$N$_{2}$ headspace as "labelled" gradients for simplicity and gradients with $^{14}$N$_{2}$ headspace incubation derived DNA as "control" gradients".

We used an RNA-Seq differential expression statistical framework to find OTUs enriched in heavy fractions of labelled gradients relative to corresponding density fractions in control gradients (for review of RNA-Seq differential expression statistics applied to microbiome OTU count data see McMurdie et al. (2014)). We use the term “differential abundance” (coined by McMurdie et al. (2014)) to denote OTUs that have different proportion means across sample classes (in this case the only sample class is labelled/control gradient). CsCl gradient fractions were categorized as "heavy" or "light". The heavy category denotes fractions with density values above X.XXXX g/mL. Since we are only interested in enriched OTUs (labelled:control) we used a one-sided Wald test for differential abundance (DOUBLE CHECK THIS AND DESCRIBE NULL AND L2FC THRESHOLD). We used DESeq2 to calculate proportion means within the sample classes and corresonding variances CITE AND PARAMETERS. The ratio of labbelled gradient heavy fraction proportion mean to corresponding control mean is expressed here as log$_{2}$ of the mean ratio (labelled:control) and is subject to shrinkage based on the information content of the OTU (see CITE for details). The mean ratio shrinkage allows for reliable ranking of mean ratios such that high variance, likely statistically insignificant mean ratios are appropriately shrunk and subsequently ranked lower than they would be if ranked by raw mean ratios. To summarize, OTUs with high labbeled:control mean ratios have higher relative abundance values in heavy fractions of labelled gradients relative to values in heavy fractions of control gradients and therefore have likely incorporated atmospheric N into their DNA during  the incubation and are putative diazotrophs.
%Here we use log2 of the proportion mean ratio (means are derived from OTU %proportions for all samples in each given class) as our differential abundance metric. It is also important to note that %the DESeq2 R package we are using to calculate the differential abundance metric “shrinks” the metric in %inverse proportion to the information content for each OTU. In this way the magnitude of the differential %abundance metric will be high only for OTUs which we have strong confidence of true differential abundance and %%the metric can be used to effectively rank OTUs by magnitude of the sample class affect (i.e. OTUs with high %proportion mean differences but also high within sample class proportion variance will not produce misleadingl %large differential abundance metric values). The DESeq2 RNA-Seq statistical framework has been shown to improve %power and specificity when identifying differentially abundant OTUs across sample classes in microbiome %experiments (McMurdie 2014).

\subsubsection{Ordination}
Principal coordinate ordinations depict the relationship between samples at each time point (day 2 and 4). Weighted Unifrac distances \cite{16332807} were used as the sample distance metric for ordination. The tree used in the unifrac distance calculations is described above. The Phyloseq \cite{24699258} wrapper for Vegan \cite{vegan} (both R packages) was used to compute sample values along principal coordinate axes. GGplot2 \cite{ggplot2} was used to display sample points along the first and second principal axes.    