\section{Methods}
\subsection{Field sites}
\subsection{Soil crust incubation}
\subsection{DNA extraction}
DNA from each sample was extracted using a MoBio PowerSoil DNA Isolation Kit (following manufacturer’s protocol, but substituting a 2 minute bead beating for the vortexing step), and then gel purified. Extracts were quantified using PicoGreen nucleic acid quantification dyes (Molecular Probes). 
\subsection{DNA-SIP}
 Gradient density centrifugation of DNA was undertaken in 6 mL polyallomer centrifuge tubes in a TLA-110 fixed angle rotor (both Beckman Coulter) in CsCl gradients with an average density of 1.725 g mL-1.  Average density for all prepared gradients was checked with an AR200 refractometer before runs. Between 2.5- 5 μg of DNA extract was added to the CsCl solution, and gradients were run under conditions of 20°C for 67 hours at 55,000 rpm (Lueders et al., 2004). Centrifuged gradients were fractionated from bottom to top in 36 equal fractions of 100 μL, using a displacement technique similar to Manefield et al. (2002). The density of each fraction was determined using a refractometer. DNA in each fraction was desalted through four washes with 300 μL TE per fraction. 
\subsection{PCR, library normalization and DNA sequencing}
Bacterial and archaeal 16s rRNA genes from each fraction were quantified through real-time PCR, using primers Ba519f/Ba907r (Stubner, 2002).  Each 25 μL reaction contained 1X Quantifast SYBR Green Master Mix (Qiagen), 0.3 μM of each primer, and 1 μL of a 1:100 dilution of fraction DNA. Thermal cycling occurred with an initial denaturation step of 10 minutes at 95°C, followed by 40 cycles of amplification (15s at 95°C, then 60s at 60°C). After each run, a melt curve was measured and recorded between 60°C and 95°C. Quantification was achieved through use of a dilution series (108-101 copies/μL) from nearly full length 16s rRNA gene amplicons from pure culture DNA of K. pneumoniae. 

Barcoded PCR of bacterial and archaeal 16s rRNA genes, in preparation for 454 Pyrosequencing, was carried out using primer set 515F/806R \cite{21349862}. The primer 806R contained an 8 bp barcode sequence, a "TC" linker, and a Roche 454 B sequencing adaptor, while the primer 515F contained the Roche 454 A sequencing adapter. Each 25 $\mu$L reaction contained 1x PCR Gold Buffer (Roche), 2.5 mM MgCl2, 200 $\mu$M of each of the four dNTPs (Promega), 0.5 mg/mL BSA (New England Biolabs), 0.3 $\mu$M of each primers, 1.25 U of Amplitaq Gold (Roche), and 8 $\mu$L of template. Template for each sample was added at normalized amounts in an attempt to prevent chimera formation, and each sample was amplified in triplicate. Thermal cycling occurred with an initial denaturation step of 5 minutes at 95°C, followed by 40 cycles of amplification (20s at 95°C, 20s at 53°C, 30s at 72°C), and a final extension step of 5 min at 72°C. Triplicate amplicons were pooled and purified using Agencourt AMPure PCR purification beads, following manufacturer’s protocol. Once cleaned, amplicons were quantified using PicoGreen nucleic acid quantification dyes (Molecular Probes) and pooled together in equimolar amounts. Samples were sent to the Environmental Genomics Core Facility at the University of South Carolina (now Selah Genomics) to be run on a Roche FLX 454 pyrosequencing machine. 

\subsection{Data Analysis} 

\subsubsection{Sequence Quality Control} Sequences were initially screened by maximum expected errors at a specific read length threshold \cite{23955772} which has been shown to be as effective as denoising 454 reads with respect to removing pyrosequencing errors. Specifically, reads were first truncated to 230 nt (all reads shorter than 230 nt were discarded) and any read that exceeded a maximum expected error threshold of 1.0 was removed. After truncation and max expected error trimming, 91\% of original reads remained. The first 30 nt representing the forward primer and barcode on high quality, truncated reads were trimmed. Remaining reads were taxonomically annotated using the "UClust" taxonomic annotation framework in the QIIME software package \cite{20383131, 20709691} with cluster seeds from Silva SSU rRNA database \cite{17947321} 97\% sequence identity OTUs as reference (release 111Ref). Reads annotated as "Chloroplast", "Eukaryota", "Archaea", "Unassigned" or "mitochondria" were culled from the dataset. Finally, reads were aligned to the Silva reference alignment provided by the Mothur software package \cite{19801464} using the Mothur NAST aligner \cite{16845035}. Allreads that did not appear to align to the expected amplicon region of the SSU rRNA gene were discarded. Quality control parameters removed 34716 of 258763 raw reads.

\subsubsection{Sequence Clustering}
pass