\section{Materials and Methods}
\subsection{BSC sampling and incubation conditions}
DNA was extracted from 1 g of BSC. Samples were taken from Green Butte, Arizona
as previously described (site CP3, \citet{BERALDI_CAMPESI_2009}). All samples
were from early successional crusts as described by \cite{15643930}.  Early
successional BSC samples (37.5 cm$^{2}$, average mass 35 g) were incubated in
sealed chambers under controlled atmosphere and in the light for 4 days. Crusts
were sampled and transported while dry and wetted at initiation of the
experiment. Treatments included an unlabeled control air headspace and
$^{15}$N$_{2}$ enriched air ($>$98\% atom $^{15}$N$_{2}$) headspace. Samples
were taken at 2 days and 4 days incubation. Acetylene Reduction rates were
measured daily. Acetylene reduction rates increased over the course of the
experiment (0.8, 4.8, 8.8, and 14.5 $\mu$m m-2 hr-1 ethylene for days 1 through
4, respectively). 

\subsection{DNA extraction}
DNA from each sample was extracted using a MoBio PowerSoil DNA Isolation Kit
(following manufacturer’s protocol, but substituting a 2 minute bead beating
for the vortexing step), and then gel purified to select high molecular 
weight DNA (replace_contentgt;$4 kb) using a 1\% low melt agarose gel and $\beta$-agarase I for
digestion (manufacturer's protocol, NEB, M0392S).  Extracts were quantified using
PicoGreen nucleic acid quantification dyes (Molecular Probes).

\subsection{DNA-SIP} CsCl density gradients were formed in 4.7 mL polyallomer
centrifuge tubes filled with gradient buffer (15mM Tris-HCl, pH 8; 15mM EDTA;
15mM KCl) which contained 1.725 g ml-1 CsCl. CsCl density was checked with a
digital refractometer as described below. A total of 2.5-5 ug of DNA was added
to each tube, and the tubes mixed, prior to centrifugation. Centrifugation was
performed in a TLA-110 fixed angle rotor (Beckman Coulter) at 20C for 67 hours
at 55,000 rpm. \citep{17369332}. Centrifuged gradients were fractionated from
bottom to top in 36 equal fractions of 100 $\mu$L, using a by syringe pump as
described Manefield et al. (2002). The density of each fraction was determined
using using an AR200 refractometer modified to accomidate 5ul samples as 
described previously \citep{17369332}. DNA in each fraction was desalted on a
filter plate (PALL, AcroPrep Advance 96 Filter Plate, Product Number 8035),
using four washes with 300$\mu$L TE per fraction. After each wash, the filter
plate was centrifuges at 500 g for 10 minutes, with a final spin of 20 minutes.
Fractions were resuspended in 50 uL of TE buffer.

 \subsection{PCR, library normalization and DNA sequencing} 
 Barcoded PCR of bacterial and archaeal 16S
 rRNA genes, in preparation for 454 Pyrosequencing, was carried out using
 primer set 515F/806R \citep{21349862} (primers purchased from Integrated DNA
 Technologies).  The primer 806R contained an 8 bp
 barcode sequence, a "TC" linker, and a Roche 454 B sequencing adaptor, while
 the primer 515F contained the Roche 454 A sequencing adapter. Each 25 $\mu$L
 reaction contained 1x PCR Gold Buffer (Roche), 2.5 mM MgCl$_{2}$, 200 $\mu$M
 of each of the four dNTPs (Promega), 0.5 mg/mL BSA (New England Biolabs), 0.3
 $\mu$M of each primers, 1.25 U of Amplitaq Gold (Roche), and 8 $\mu$L of
 template. Each sample was amplified in triplicate. Thermal cycling occurred
 with an initial denaturation step of 5 minutes at 95°C, followed by 40 cycles
 of amplification (20s at 95°C, 20s at 53°C, 30s at 72°C), and a final
 extension step of 5 min at 72°C. Triplicate amplicons were pooled and purified
 using Agencourt AMPure PCR purification beads, following manufacturer’s
 protocol. Once purified, amplicons were quantified using PicoGreen nucleic acid
 quantification dyes (Molecular Probes) and pooled together in equimolar
 amounts. Samples were sent to the Environmental Genomics Core Facility at the
 University of South Carolina (now Selah Genomics) to be run on a Roche FLX 454
 pyrosequencing machine (FLX-Titanium platform). 

\subsection{Data analysis} 
\subsubsection{Sequence quality control}
Sequences were initially screened by maximum expected errors at a specific read
length threshold \citep{23955772} which has been shown to be as effective as
denoising with respect to removing pyrosequencing errors. Specifically, reads
were first truncated to 230 nucleotides (nt) (all reads shorter than 230 nt
were discarded) and any read that exceeded a maximum expected error threshold
of 1.0 was removed. After truncation and max expected error trimming, 91\% of
original reads remained.  Forward primer and barcode was then removed from the
high quality, truncated reads.  Remaining reads were taxonomically annotated
using the "UClust" taxonomic annotation framework in the QIIME software package
\citep{20383131, 20709691} with cluster seeds from Silva SSU rRNA database
\citep{17947321} 97\% sequence identity OTUs as reference (release 111Ref).
Reads annotated as "Chloroplast", "Eukaryota", "Archaea", "Unassigned" or
"mitochondria" were culled from the dataset. Finally, reads were aligned to the
Silva reference alignment provided by the Mothur software package
\citep{19801464} using the Mothur NAST aligner \citep{16845035}. All reads that
did not align to the expected amplicon region of the SSU rRNA gene were
discarded. Quality control parameters removed 34,716 of 258,763 raw reads.

\subsubsection{Sequence clustering}
Sequences were distributed into OTUs using the UParse methodology
\citep{23955772}. Specifically, cluster seeds were identified using USearch on
non-redundant reads sorted by count. The sequence
identity threshold for establishing a new OTU centroid was 97\%. After initial
cluster centroid selection, select 16S rRNA gene sequences from \citet{Yeager}
were added to the centroid collection. Specifically, \citet{Yeager} Colorado
Plateau or Moab, Utah sequences were added which included the 16S rRNA gene
sequences for \textit{Calothrix} MCC-3A (accession DQ531700.1), \textit{Nostoc
commune} MCT-1 (accession DQ531903), \textit{Nostoc commune} MFG-1 (accession
DQ531699.1), \textit{Scytonema hyalinum} DC-A (accession DQ531701.1),
\textit{Scytonema hyalinum} FGP-7A (accession DQ531697.1), \textit{Spirirestis
rafaelensis} LQ-10 (accession DQ531696.1).  Centroid sequences that matched
selected \citet{Yeager} sequences with greater than to 97\% sequence identity
were subsequently removed from the centroid collection. With USearch/UParse,
potential chimeras are identified during OTU centroid selection and are not
allowed to become cluster centroids effectively removing chimeras from the read
pool. All quality controlled reads were then mapped to cluster centroids at an
identity threshold of 97\% again using USearch. A total of 95.6\% of quality
controlled reads could be mapped to centroids. Unmapped reads do not count
towards sample counts and are removed from downstream analyses. The
USearch software version for cluster generation was 7.0.1090.
\citet{Garcia_Pichel_2013} and \citet{Steven_2013}) sequences were quality
screened by determining if they covered the expected region of the 16S rRNA
gene (described above) and included as input to USearch for OTU centroid
selection and subsequent mapping to OTU centroids. 

\subsubsection{Phylogenetic tree}
The alignment for the "\textit{Clostridiaceae}" phylogeny was created using
SSU-Align which is based on Infernal \citep{24008419, 19307242}. Columns in the
alignment that were not included in the SSU-Align covariance models or were
aligned with poor confidence (less than 95\% of characters in a position had
posterior probability alignment scores of at least 95\%) were masked for
phylogenetic reconstruction. Additionally, the alignment was trimmed to
coordinates such that all sequences in the alignment began and ended at the
same positions. The "\textit{Clostridiaceae}" tree included all top BLAST hits
(parameters below) for $^{15}$N \textit{Clostridiaceae} responders in the
Living Tree Project database \citep{Yarza_2008} in addition to BLAST hits
within a sequence identity
threshold of 97\% to $^{15}$N responders from the Silva SSURef\_NR SSU rRNA
database \citep{17947321}. Only one SSURef\_NR115 hit per study per OTU
("study" was determined by "title" field) was selected for the tree. FastTree
\citep{20224823} was used to build the tree and support values are
SH-like scores reported by FastTree. Short sequences were mapped to the
reference backbone using pplacer \citep{Matsen_2010} (default parameters).
pplacer finds the edge placements that maximize phylogenetic likelihood. Prior
to being mapped to the reference tree, short sequences were aligned to the
reference alignment using Infernal \citep{19307242} against the same SSU-Align
covariance model used to align reference sequences.

\subsubsection{Identifying OTUs that incorporated $^{15}$N into their DNA}
DNA-SIP is a culture-independent approach towards defining identity-function
connections in microbial communities \citep{Buckley_2011, 17446886, Radajewski_2001}. Microbes
participating in a specific process are identified on the basis of isotope
assimilation into DNA. Isotopically labeled nucleic acids can be
separated from unlabeled by buoyant density in a CsCl gradient. As the
buoyant density of a macromolecule is dependent on many factors in addition
to stable isotope incorporation (e.g. GC-content in nucleic acids
\citep{25139123}), labeled nucleic acids from one microbial population may
have the same buoyant density of unlabeled nucleic acids from another.
Therefore it is imperative to compare density gradients with nucleic acids
from heavy stable isotope incubations to gradients from ``control''
incubations where everything mimics the experimental conditions except that
unlabeled substrates are used. By contrasting heavy density gradient
fractions in experimental density gradients (hereafter referred to as "labeled"
gradients) against heavy fractions in control gradients, the identities of
microbes with labeled nucleic acids can be determined 

We used an RNA-Seq differential expression statistical framework
\citep{Love_2014} to find OTUs enriched in heavy fractions of labeled
gradients relative to corresponding density fractions in control gradients
(for review of RNA-Seq differential expression statistics applied to
microbiome OTU count data see \citet{24699258}). We use the term
“differential abundance” (coined by \citet{24699258}) to denote OTUs that
have different proportion means across sample classes (in this case the only
sample class is labeled/control).  CsCl gradient fractions were categorized
as "heavy" or "light". The heavy category denotes fractions with density
values above 1.725 g/mL. Since we are only interested in enriched OTUs
(labeled versus control), we used a one-sided z-test for differential
abundance (the null hypothesis is the labeled:control proportion mean ratio
for an OTU is less than a selected threshold). P-values were corrected with
the Benjamini and Hochberg method \citep{citeulike:1042553}. We selected a
log$_{2}$ fold change null threshold of 0.25 (or a labeled:control proportion
mean ratio of 1.19). DESeq2 was used to calculate the moderated log$_{2}$
fold change of labeled:control proportion mean ratios and corresponding
standard errors. Mean ratio moderation allows for reliable ratio ranking such
that high variance and likely statistically insignificant mean ratios are
appropriately shrunk and subsequently ranked lower than they would be as raw
ratios. Those OTUs that exhibit a statistically significant increase in
proportion in heavy fractions from $^{15}$N$_{2}$-labeled samples relative to
corresponding controls have increased significantly in bouyant density in
response to $^{15}$N$_{2}$ treatment; a response that is expected for
N$_{2}$-fixing organisms.

\subsubsection{Community and Sequence Analysis}
BLAST searches were done with the "blastn" program from BLAST+ toolkit
\citep{20003500} version 2.2.29+. Default parameters were always employed and
the BioPython \citep{19304878} BLAST+ wrapper was used to invoke the blastn
program. Pandas \citep{citeulike:11241428} and dplyr \citep{dplyr} were used to
parse and munge BLAST output tables.  

Principal coordinate ordinations depict the relationship between
samples at each time point (day 2 and 4). Bray-Curtis distances were
used as the sample distance metric for ordination. The Phyloseq
\citep{24699258} wrapper for Vegan \citep{vegan} (both R packages) was
used to compute sample values along principal coordinate axes. GGplot2
\citep{ggplot2} was used to display sample points along the first and
second principal axes. Adonis tests \cite{Anderson_2001} were done with default
number of permutations (1000).

Rarefaction curves were created using
bioinformatics modules in the PyCogent Python package \citep{Knight_2007}.
Parametric richness estimates were made with CatchAll using only the best model
for total OTU estimates \citep{BUNGE_2010}.

All code to take raw sequencing data through the presented figures can be
found at:

\url{http://nbviewer.ipython.org/github/chuckpr/NSIP_data_analysis}

