\section{Results}

\subsection{Comparison of sequence collections at "study"-level}

\subsubsection{Comparisons of OTU content}
Of the 4340 OTU centroids established for this study (including sequences from \citet{Steven_2013} and \cite{Garcia_Pichel_2013}) 445 and 870 have matches in the Living Tree Project (LTP) (a collection of 16S gene sequences for all sequenced type strains \cite{Yarza_2008}) at greater or equal than 97\% and 95\% sequence identity, respectively (LTP version 115). Similar numbers of total OTUs were found in each data set explored in this study (i.e. the DNA-SIP data presented here, the data presented by \citet{Steven_2013} and by \citet{Garcia_Pichel_2013}). Specifically, there were 3079 OTUs (209,354 total sequences after quality control) in the DNA-SIP data, 3203 OTUs (129,033 total sequences after quality control) in the \citet{Garcia_Pichel_2013} study, and 2481 OTUs (129,358 total sequences after quality control) in the \citet{Steven_2013} study. The DNA-SIP data set shares more OTUs with the \citet{Steven_2013} (56\% of total OTUs found in either of the two data sets) than it does with the \citet{Garcia_Pichel_2013} data (46\% of total OTUs between both data sets). The \citet{Steven_2013} and \citet{Garcia_Pichel_2013} only share 46\% of OTUs.

\subsubsection{Comparisons of Taxonomic Content} 
\textit{Cyanobacteria} and \textit{Proteobacteria} were the top two phylum-level sequence annotations for all three studies but only the DNA-SIP data had more \textit{Proteobacteria} annotations than \textit{Cyanobacteria}. \textit{Proteobacteria} represented the 29.8\% of sequence annotations in DNA-SIP data as opposed to 17.8\% and 19.2\% for the \citet{Garcia_Pichel_2013} and \citet{Steven_2013} data, respectively. Figure X shows the distribution of phylum-level sequence annotations for each study in the nine most abundant  phyla across all studies, as determined by raw sequence counts. There is a stark contrast in the total percentage of sequences annotated as \textit{Firmicutes} between the raw environmental samples and the DNA-SIP data. \textit{Firmicutes} represent only 0.21\% and 0.23\% of total phylum level sequence annotations in the \citet{Steven_2013} and \citet{Garcia_Pichel_2013} studies, respectively. In the DNA-SIP sequence collection \textit{Firmicutes} make up 19\% of phylum level sequence annotations. Also in sharp contrast for the DNA-SIP versus environmental data is the number of putative heterocystous \textit{Cyanobacteria} sequences. Only 0.29\% of \textit{Cyanobacteria} sequences in the DNA-SIP data are annotated as belonging to "Subsection IV" which is the heterocystous order of \textit{Cyanobacteria} in the Silva taxonomic nomenclature \cite{17947321}. In the \citet{Steven_2013} and \citet{Garcia_Pichel_2013} studies 15\% and 23\%, respectively, of \texit{Cyanobacteria} sequences are annotated as belonging to "Subsection IV".  

\subsection{Ordination of CsCl gradient fraction SSU rRNA libraries}
Ordination of Bray-Curtis \cite{Bray_1957} distances between CsCl gradient fraction sequence libraries with principal coordinates analysis shows the labeled gradient fraction libraries diverge from control in the heavy fractions (Figure X). When the labeled and control gradient fractions are paired such that each pair contains a control fraction and labeled fraction with a density difference below XXX g/mL, the Bray-Curtis distance between the fraction pair is postively correlated to the density of the labeled fraction (p-value: 0.00052, r$^{2}$: 0.3315) (Figure X). Additionally, the label/control category for heavy fractions is statistically significant by the Adonis test (p-value: 0.001, r$^{2}$: 0.136) \cite{Anderson_2001}. The first principal axis appears to be correlated with fraction density (Adonis test p-value for density with all CsCl fraction libraries: 0.001, r$^{2}$ 0.117).

\subsection{Identities of possible $^{15}$N incorporators}
The OTUs that have proportion means in heavy fractions that are enriched in labeled versus control gradients are those that have responded to the stable isotope tracer, which would indicate diazotropy in this experiment. We found 38 responders total using a false discovery rate threshold for multiple comparison adjusted p-values of 10\%. Of these 38, 26 are annotated as \textit{Firmicutes}, 9 as \textit{Proteobacteria}, 2 as \textit{Acidobacteria} and 1 as \textit{Actinobacteria} (The inset of Figure X summarizes the Family level taxanomic profile of stable isotope responders). Figure X summarizes the ratio of proportion means for each OTU where means are calculated from proportions in heavy fractions within labeled or controlled gradients and the ratio is labeled over control (see methods). If the OTUs are ranked by descending, moderated proportion mean labeled:control ratios, the top 10 ratios (i.e. the 10 OTUs that were most enriched in the labeled gradients in heavy fractions) are either \textit{Firmicutes} (6 OTUS) or \textit{Proteobacteria} (4 OTUs). Table X summarizes the results from BLAST searching the centroid sequences for these top 10 OTUs against the LTP (version 115). The \textit{Proteobacteria} OTU centroid seqeunces for the top 10 responders all share high identity ($>$98.48\% identity, Table X) with cultivars from genera known to possess diazotrophs including \textit{Klebsiella}, \textit{Shigella}, \textit{Acinetobacter}, and \textit{Ideonella}. None of the \textit{Firmicutes} OTUs in the top 10 responders share greater than 97\% sequence identity with sequences in the LTP (relase 115) (see Table X).

\subsection{Distribution of BSC Diazotrophs in Environmental Samples}
\subsubsection{Non-Cyanobacterial Taxa}
\paragraph{\textit{Clostridiacea}}
Five of the 6 \textit{Firmicutes} in the top 10 responder OTUs belong in the \textit{Clostridiacea}. We only observed one of these strongly responding \textit{Clostrideacea} in the data presented by \citet{Garcia_Pichel_2013}, "OTU.108" (closest BLAST hit in LTP Relase 115 -- \textit{Caloramotor proteoclasticus}, BLAST \%ID 96.94, Accession X90488). OTU.108 was found in two samples both characterized as "light" crust. One other \textit{Clostrideacea} OTU with a proportion mean ratio (labeled:control) p-value less than 0.10 but outside the top 10 responders was found in the \citet{Garcia_Pichel_2013} data and also in a "light" crust sample. None of the strongly responding \textit{Clostridiacea} were found in the sequences provided by \citet{Steven_2013}.

Figure X depicts the phylogenetic breadth of \textit{Clostridiaceae} N responder OTUs from this experiment. The phylogenetic tree was constructed from nearly full-length reference sequences, and edge width demonstrates the placements of short OTU centroid sequences in the backbone tree (see methods SECTION for description of placement algorithm and selection criteria for reference sequences). As shown, \textit{Clostridiaceae} N-responder OTU centroid 16S sequences are generally more closely related to environmental than cultivar 16S gene sequences.   

\paragraph{\textit{Gammaproteobacteria}}
Only "OTU.342" (closest BLAST hit in LTP Release 115, BLAST \%ID 100, Accession ZD3440, \textit{Acinetobacter johnsonii}) of the \textit{Proteobacteria} OTUs in the top 10 most strongly responding OTUs was found in the \citet{Garcia_Pichel_2013} sequences. None of the strongly responding \textit{Protebacteria} OTUs were found in the \citet{Steven_2013} sequences. There were 133 responder OTU-sample occurrences (SIP responding OTU was found in a sample library) in the \citet{Steven_2013} data. 83 were in "below crust" samples, 50 in BSC samples.
\paragraph{Other taxa} 
Two OTUs were found in 61 of 65 samples from the combined data sets of \citet{Garcia_Pichel_2013} and \citet{Steven_2013}. Both OTUs were annotated as \textit{Acidobacteria} but shared little sequence identity to any cultivar SSU rRNA gene sequences in the LTP (Release 115), with best LTP BLAST hits of 81.91 and 81.32 \% identity. Additionally, the evidence for N incorporation for each OTU was weak relative to other putative responders (adjusted p-values of 0.090 and 0.096). Of the remaining 36 stable isotope responder OTUs, only 14 were observed in the environmental data. Figure X summarizes the OTU-sample occurrences in both the \citet{Steven_2013} and the \citet{Garcia_Pichel_2013} data with occurrences distributed into the most relevant sample classes of each respective study.

\subsubsection{Heterocystous \textit{Cyanobacteria}}
At least one of the six OTUs defined by sequences reported by \citet{Yeager} (see Table~\ref{table:yeager_2006}) was found in 21 of the 23 sites surveyed by \citet{Garcia_Pichel_2013}. OTUs defined by \textit{Scytonema hyalinum} FGP-7A and \textit{Scytonema hyalinum} DC-A 16S rRNA gene sequences were found in 18 and 17 sites, respectively. \textit{Nostoc commune} MCT-1 and \textit{Spirirestis rafaelensis} LQ-10 defined OTUs we each found in 16 sites. The OTU defined by \textit{Nostoc commune} MFG-1 was found in 12 sites and the OTU defined by \textit{Calothrix} MCC-3A was only found in one site surveyed by \citet{Garcia_Pichel_2013}. The opposite BSC relative abundance relationships of \textit{Microcoleus Vaginatus} and \textit{M. Strenstrupii} with site mean annual temperature was a major finding by \citet{Garcia_Pichel_2013}. \citet{Garcia_Pichel_2013} did not report the relationship of diaztrophic cyanobacteria with temperature although a comment by \citet{Belnap28062013} briefly discusses a qualitative positive relationship of \textit{Scytonema} with temperature in the \citet{Garcia_Pichel_2013} data. In agreement with the \citet{Belnap28062013} interpretation we found a positive relationship of \textit{Scytonema hyalinum} FGP-7A OTU relative abundance with mean annual temperature (p-value XXXXX) but the trend is not as clear for \textit{Scytonema hyalinum} DC-A (Figure X). We also found \textit{Nostoc commune} MCT-1 and MFG-1 OTU relative abundance was inversely related to mean annual temperature (p-values XXXX and XXXX, respectively) (Figure X). 

At least one OTU defined by selected 16S rRNA gene sequences presented by \citet{Yeager} (Table~\ref{table:yeager_2006}) was found in all but 7 of 42 samples surveyed by \citet{Steven_2013} and all of these 7 were "below crust" samples. Table X summarizes the distribution of \citet{Yeager} sequence defined OTUs in \citet{Steven_2013} samples. As expected all of the six OTUs defined by \citet{Yeager} sequences were more abundant in the crust samples than below crust samples (Figure X) (maximum p-value for any OTU: 1.96x10$^{-4}$).

\subsection{Richness estimates}
Rarefaction curves were created using bioinformatics modules in the PyCognet Python package \cite{Knight_2007}. Parametric richness estimates were made with CatchAll using Only the best model total OTU estimates \cite{BUNGE_2010}.