\section{Results}
\subsection{Ordination of CsCl gradient fraction SSU rRNA libraries} 
BSC were incubated for 4 days in the presence or absence of $^{15}$N$_{2}$ and
DNA was extracted for DNA-SIP at 2 and 4 days. Fractionation of CsCl gradients
permitted separation of DNA on the basis of buoyant density. Ordination of
Bray-Curtis \citep{Bray_1957} distances beteen SSU-rRNA amplicon sequence
collections from gradient fractions reveals that labeled gradient fraction
(i.e. gradient fractions of DNA from $^{15}$N$_{2}$ incubations) sequence
collections diverge from control (i.e. DNA from incubations without
$^{15}$N$_{2}$) at the ``heavy`` of the CsCl gradients
(Figure~\ref{fig:ordination}). Although the density position of gradient
fractions from different gradients do not match perfectly, fraction pairs from
corresponding control versus labeled gradients can be constructed by pairing
control gradient fractions with their closest density neighbors/fractions from
corresponding labeled gradients.  If a gradient fraction did not have a mate
within a density difference of 0.003 g/mL it remained unpaired. Bray-Curtis
distance between the fraction pairs is positively correlated to the density of
the labeled fraction (p-value: 0.00052, r$^{2}$: 0.3315) (inset
Figure~\ref{fig:ordination}). Additionally, differences among label/control groups
with heavy fractions are statistically significant by the Adonis test (p-value:
0.001, r$^{2}$: 0.136) \citep{Anderson_2001}. The first principal axis appears
to be correlated with fraction density (Figure~\ref{fig:ordination}) and the
Adonis test p-value for density versus pairwise Bray-Curtis distances with all
CsCl fraction libraries is 0.001 (r$^{2}$ 0.117).

\subsection{Identities of OTUs responsive to $^{15}$N$_{2}$} 
A statistically significant increase in OTU abundance in heavy fractions of
$^{15}$N$_{2}$ labeled samples relative to corresponding gradient fractions
from controls provides evidence for OTUs that have incorporated $^{15}$N into
their DNA. Specifically, we compared OTU proportion means between labeled and
control samples from heavy gradient fractions using statistics developed to
find differentially expressed genes with RNASeq data (CITE McMurdie and
DESeq2). p-values were adjusted by the BH method CITE and we used a false
discovery rate (FDR) cutoff of 0.10 (typical FDR threshold in gene expression
data analysis, CITE DESeq2) to reject the null hypothesis that labeled versus
control proportion mean differences were below a chosen threshold (see
methods). With the above methods 38 OTUs had labeled versus control proportion
mean difference p-values below 0.10 for one or both incubation days. These OTUs
likely to incorporated $^{15}$N into DNA ($^{15}$N$_{2}$ ``responders''). Of
these 38, 26 are annotated as \textit{Firmicutes}, 9 as
\textit{Proteobacteria}, 2 as \textit{Acidobacteria} and 1 as
\textit{Actinobacteria} (Figure~\ref{fig:l2fc}). If the OTUs are ranked by
descending, moderated proportion mean labeled:control ratios, the top 10 ratios
(i.e. the 10 OTUs that were most enriched in the labeled gradients considering
only heavy fractions) are either \textit{Firmicutes} (6 OTUs) or
\textit{Proteobacteria} (4 OTUs) (Figure~\ref{fig:scatter_heavy}).
\textit{Proteobacteria} OTU centroid sequences for the top 10 responders all
share high identity ($>$98.48\% identity, Table~\ref{tab:LTP_blast}) with
cultivars from genera known to possess diazotrophs including
\textit{Klebsiella}, \textit{Shigella}, \textit{Acinetobacter}, and
\textit{Ideonella}. None of the \textit{Firmicutes} OTUs in the top 10
responders share greater than 97\% sequence identity with sequences in the LTP
database (release 115) (see Table~\ref{tab:LTP_blast}).

\subsection{Comparison of sequence collections at "study"-level}
\subsubsection{Comparisons of OTU content:} There were 3,079 OTUs (209,354
total sequences after quality control) in the DNA-SIP data, 3,203 OTUs (129,033
total sequences after quality control) in the \citet{Garcia_Pichel_2013} study,
and 2,481 OTUs (129,358 total sequences after quality control) in the
\citet{Steven_2013} study. Of the 4,340 OTU centroids established for this
study (including sequences from \citet{Steven_2013} and
\citet{Garcia_Pichel_2013}) 445 have matches in the Living Tree Project (LTP)
(a collection of 16S gene sequences for all sequenced type strains
\citep{Yarza_2008}) at greater or equal than 97\% (LTP version 115). That is,
445 of 4,340 are closely related to cultivars. The DNA-SIP data set shares 56\%
OTUs with the \citet{Steven_2013} data and 46\% of OTUs with the
\citet{Garcia_Pichel_2013} data (where total OTUs are from the combined data
for each pairwise comparison).  The \citet{Steven_2013} and
\citet{Garcia_Pichel_2013} share 46\% of OTUs.

\subsubsection{Comparisons of Taxonomic Content:} \textit{Cyanobacteria} and
\textit{Proteobacteria} were the top two phylum-level sequence annotations
for all three studies but only the DNA-SIP data had more
\textit{Proteobacteria} annotations than \textit{Cyanobacteria}.
\textit{Proteobacteria} represented the 29.8\% of sequence annotations in
DNA-SIP data as opposed to 17.8\% and 19.2\% for the \citet{Garcia_Pichel_2013}
and \citet{Steven_2013} data, respectively.  There is a stark contrast in the
total percentage of sequences annotated as \textit{Firmicutes} between the raw
environmental samples and the DNA-SIP data. \textit{Firmicutes} represent only
0.21\% and 0.23\% of total phylum level sequence annotations in the
\citet{Steven_2013} and \citet{Garcia_Pichel_2013} studies, respectively
(Figure~\ref{fig:study_phy_dist}). In the DNA-SIP sequence collection
\textit{Firmicutes} make up 19\% of phylum level sequence annotations. Also in
sharp contrast for the DNA-SIP versus environmental data is the number of
putative heterocystous \textit{Cyanobacteria} sequences. Only 0.29\% of
\textit{Cyanobacteria} sequences in the DNA-SIP data are annotated as belonging
to "Subsection IV" which is the heterocystous order of \textit{Cyanobacteria}
in the Silva taxonomic nomenclature \citep{17947321}. In the
\citet{Steven_2013} and \citet{Garcia_Pichel_2013} studies 15\% and 23\%,
respectively, of \textit{Cyanobacteria} sequences are annotated as belonging to
"Subsection
IV".  

\subsection{Distribution of BSC Diazotrophs in Environmental Samples}
\subsubsection{\textbf{\textit{Clostridiacea}:}} Five of the 6
\textit{Firmicutes} in the top 10 responder OTUs (above) belong in the
\textit{Clostridiacea}. We only observed one of these strongly responding
\textit{Clostridiaceae} in the data presented by \citet{Garcia_Pichel_2013},
"OTU.108" (closest BLAST hit in LTP Release 115 -- \textit{Caloramotor
proteoclasticus}, BLAST \%ID 96.94, Accession X90488).  OTU.108 was found in
two samples both characterized as "light" crust. One other
\textit{Clostridiaceae} OTU with a proportion mean ratio (labeled:control)
p-value less than 0.10 but outside the top 10 responders was found in the
\citet{Garcia_Pichel_2013} data (a "light" crust sample). None of
the strongly responding \textit{Clostridiacea} were found in the sequences
provided by \citet{Steven_2013}. \textit{Clostridiaceae} $^{15}$N-responder OTU
centroid 16S sequences are generally more closely related to environmental than
cultivar 16S gene sequences.   

\subsubsection{\textbf{\textit{Proteobacteria}:}} One of the
\textit{Proteobacteria} OTUs in the 10 most strongly responding OTUs was
found in the \citet{Garcia_Pichel_2013} sequences (closest
BLAST hit in LTP Release 115, BLAST \%ID 100, Accession ZD3440,
\textit{Acinetobacter johnsonii}). None of the strongly responding
\textit{Protebacteria} OTUs were found in the \citet{Steven_2013} sequences.
There were 133 responder OTU-sample occurrences (responder OTU was found in a
sample library) in the \citet{Steven_2013} data.  83 were in "below crust"
samples, 50 in BSC samples.

\subsubsection{\textbf{Other taxa:}} Two $^{15}$N-responsive OTUs were found
in an extensive number of environmental samples (61 of 65 samples from the
combined data sets of \citet{Garcia_Pichel_2013} and \citet{Steven_2013}).
Both OTUs were annotated as \textit{Acidobacteria} but shared little sequence
identity to any cultivar SSU rRNA gene sequences in the LTP (Release 115),
with best LTP BLAST hits of 81.91 and 81.32\% identity. Additionally, the
evidence for $^{15}$N incorporation for each OTU was weak relative to other
putative responders (adjusted p-values of 0.090 and 0.096). Of the remaining
36 stable isotope responder OTUs, only 14 were observed in the environmental
data.  Figure~\ref{fig:rspndr_dist} summarizes the OTU-sample occurrences in
both the \citet{Steven_2013} and the \citet{Garcia_Pichel_2013} data with
occurrences distributed into the most relevant sample classes of each study.

%\subsubsection{Heterocystous \textit{Cyanobacteria}} At least one OTU defined
%by \citet{Yeager} sequences (see Table~\ref{table:yeager_2006}) was found in
%21 of the 23 \citet{Garcia_Pichel_2013} sampling sites. Counts of samples
%with \citet{Yeager} sequence defined heterocystous \textit{Cyanobacteria}
%OTUs are summarized in Table~\ref{table:het_dist}. The opposite BSC relative
%abundance relationships of \textit{Microcoleus Vaginatus} and \textit{M.
%Strenstrupii} with site mean annual temperature was a major finding by
%\citet{Garcia_Pichel_2013}.  \citet{Garcia_Pichel_2013} did not report the
%relationship of diazotrophic \textit{Cyanobacteria} with temperature although
%a comment by \citet{Belnap28062013} briefly discusses a qualitative positive
%relationship of \textit{Scytonema} with temperature in the
%\citet{Garcia_Pichel_2013} data.  In agreement with the
%\citet{Belnap28062013} interpretation, we found a positive relationship of
%\textit{Scytonema hyalinum} FGP-7A and DC-A OTU relative abundance with mean
%annual temperature (p-values 3.332x10$^{-03}$ and 3.173x10$^{-04}$,
%respectively) (Figure~\ref{fig:het_temp}). We also found \textit{Nostoc
%commune} MCT-1 and MFG-1 OTU relative abundance was inversely related to mean
%annual temperature (p-values 1.307x10$^{-02}$ and 1.577x10$^{-06}$,
%respectively) (Figure~\ref{fig:het_temp}). 
%
%\begin{table}
%
%\textbf{
%    \refstepcounter{table}
%    \label{table:het_dist}
%    Table \arabic{table}. }{Counts of heterocystous \textit{Cyanobacteria}l OTU occurrences in \citet{Garcia_Pichel_2013} samples (n = 23) and \citet{Steven_2013} samples (n = 42)}
%
%{\centering
%\begin{tabular}{lrr}
%  \toprule
% Isolate & \citet{Garcia_Pichel_2013} & \citet{Steven_2013} \\ 
%  \midrule
%  \textit{Calothrix} MCC-3A &   1 &   6 \\ \midrule
%  \textit{Nostoc commune} MCT-1 &  16 &  23 \\ \midrule
%  \textit{Nostoc commune} MFG-1 &  12 &  23 \\ \midrule
%  \textit{Scytonema hyalinum} DC-A &  17 &  30 \\ \midrule
%  \textit{Scytonema hyalinum} FGP-7A &  18 &  27 \\ \midrule
%  \textit{Spirirestis rafaelensis} LQ-10 &  16 &  30 \\
%   \bottomrule
%\end{tabular}}{}
%\end{table}
%
%At least one \citet{Yeager} sequence defined OTU
%(Table~\ref{table:yeager_2006}) was found in 35 of 42 \citet{Steven_2013}
%samples. The 7 samples that lacked \citet{Yeager} OTUs were "below crust"
%samples.  Table~\ref{table:het_dist} summarizes the counts of
%\citet{Steven_2013} samples with \citet{Yeager} sequence defined OTUs. As
%expected all of the six OTUs defined by \citet{Yeager} sequences were more
%abundant in the crust samples than below crust samples
%(Figure~\ref{fig:het_steven}) (maximum p-value for any OTU: 1.96x10$^{-4}$).

